tasks/kyu8.py

```diff
@@ -15,3 +15,19 @@ def example(first, second):
         int: The return value.
     """
     return first + second
+
+
+def divisible_by(array, divisor):
+    """
+        Returns a list of numbers which are divisible by given number
+        Args:
+            array (list): The first parameter
+            divisor (int): The second parameter
+        Returns:
+            list: The return value
+    """
+    return_list = list()
+    for i in array:
```

Comment:
You can use list comprehension.

Reply:
Yes, but it still is correct to write it this way.